### PR TITLE
[CP-862] I want to protect my backup with secure password

### PR DESCRIPTION
**Pull request by wercix**
Jira: [CP-862]

**Description**
Password to protect backup file should contain:
- minimum 8 characters
- one capital letter
- one number
- one special character 

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-862]: https://appnroll.atlassian.net/browse/CP-862?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/renderer/utils/form-validators.ts [hunk 1]
```diff
@@ -84,3 +87,18 @@ export const emailValidator: RegisterOptions = {
   },
   required: intl.formatMessage({ id: "component.formErrorRequiredEmail" }),
 }
+
+export const backupSecretKeyValidator = {
```
Comment (dkarski) [inline packages/app/src/renderer/utils/form-validators.ts:91]: validator isn't tested

File: packages/app/src/ui/components/password-field/password-field.styled.tsx [hunk 2]
```diff
@@ -19,12 +19,12 @@ export const FormInput = styled(InputComponent)`
...
 export const FieldWrapper = styled.div`
   position: relative;
-  margin-bottom: 1.4rem;
+  margin-bottom: 3.6rem;
 `
 
 export const IconWrapper = styled.div`
   position: absolute;
-  top: 0;
+  top: -1rem;
```
Comment (dkarski) [inline packages/app/src/ui/components/password-field/password-field.styled.tsx:27]: why static value? What is the purpose of the change? Centering? What if the icon size will changes? I suggest changing the value to percent.

**Review by dkarski — APPROVED** [review]

**Review by igorbezsmertnyi — APPROVED** [review]
